cookbooks/aws-parallelcluster-install/recipes/disable_services.rb

```diff
@@ -15,13 +15,11 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
-# default openmpi installation conflicts with new install
-# new one is installed in /opt/amazon/efa/bin/
+# Disable DLAMI multi eni helper
+# no only_if statement because if the service is not present the action disable does not return error
+disable_service('aws-ubuntu-eni-helper', 'debian')
```

Comment:
would it make sense to mask the service so that if not present at AMI build but installed at a later stage it will stay disabled? also we could do the same for alinux2 in case the service gets added there too